pilot/pkg/config/memory/config.go

```diff
@@ -96,6 +109,10 @@ func (cr *store) List(typ, namespace string) ([]model.Config, error) {
 	return out, nil
 }
 
+func buildKey(typ, name, namespace string) string {
+	return fmt.Sprintf("%s/%s/%s", typ, name, namespace)
```

Comment:
nit: `%s/%s.%s` maybe be more consistent with kubernetes. Doesn't really matter though since its an implementation detail